springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/types/AsyncAPI.java

```diff
@@ -33,7 +37,7 @@ public class AsyncAPI {
      */
     @NonNull
     @Builder.Default
-    private String asyncapi = "2.0.0";
+    private String asyncapi = "2.6.0";
```

Comment:
Non Blocking Comment: While working in another change I saw that this class is mainly a duplicated of https://github.com/asyncapi/jasyncapi/blob/master/asyncapi-core/src/main/java/com/asyncapi/v2/_0_0/model/AsyncAPI.java. Maybe would be a good idea to try to use that file directly.

Reply:
Agreed. That would be a benefit to the project and resolve some of the current issues.
If I remember correctly, the difference is the Schema type. At this point we do not have a mapping of the Jackson schema into the AsyncAPI schema.
Any help or suggestions on this are welcome.